nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 15;
+
```

Comment:
> But is there an equivalent to this menu_margin thing? Does it keep a buffer of a fixed amount of minimum space between the menu and the farthest edge of the screen?

ah good call, this does press right up against the edge of the screen. It does seem like the norm.